### PR TITLE
completed avatar

### PR DESCRIPTION
**Pull request by sharmivijay**
Team,
Kindly review

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -3,8 +3,65 @@
...
+    border: 1px solid var(--white-default);;
+    background-color: var(--primary-color);
+  }
+
+.avatar-medium{
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    border: 1px solid var(--white-default);k;
+    background-color: var(--primary-color);
```
Comment (shuntzhere) [inline styles.css:47]: I think it's typo here "k"

File: styles.css [hunk 2]
```diff
@@ -3,8 +3,65 @@
...
+    column-gap: 2em;
+    text-align: center;
+}
+
+.avatar-small{
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    border: 1px solid var(--white-default);;
+    background-color: var(--primary-color);
```
Comment (shuntzhere) [inline styles.css:39]: Here's an extra semi-colon.

**Review by shuntzhere — COMMENTED** [review]
Move the svg's to gitignore.

File: styles.css [hunk 3]
```diff
@@ -3,8 +3,65 @@
...
+  }
```
Comment (shuntzhere) [inline styles.css]: Line 38, extra semi-colon.

File: styles.css [hunk 4]
```diff
@@ -3,8 +3,65 @@
...
+
+.avatar-medium{
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    border: 1px solid var(--white-default);k;
+    background-color: var(--primary-color);
+  }
```
Comment (shuntzhere) [inline styles.css]: There is typo at line 46. Remove that

File: styles.css [hunk 5]
```diff
@@ -3,8 +3,65 @@
     --primary-color: #818CF8;
     --dark-gray: #6B7280;
     --light-gray: #c9cacc;
+    --white-default: white; 
 }
```
Comment (shuntzhere) [inline styles.css]: You can move these files to a variable.css file for more clarity.

**Review by shuntzhere — COMMENTED** [review]
You can move the svg's to gitignore. And I've suggested some minor changes.

File: styles.css [hunk 6]
```diff
@@ -3,8 +3,65 @@
...
\ No newline at end of file
+}
+
+.section {
+    padding: 2rem;
+    margin: 1rem;
+}
+
+
+.non-white {
```
Comment (junaidshaikh-js) [inline styles.css]: can be renames as `bg-gray` it makes more sense

File: styles.css [hunk 7]
```diff
@@ -3,8 +3,65 @@
...
+    background-color: var(--light-gray);
+}
+
+.white{
```
Comment (junaidshaikh-js) [inline styles.css]: Here also, can be names as `bg-white`

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

**Pull request merged**